demo.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+export PREFIX=cb
+export COMPOSE_TIMEOUTDOCKER_CLIENT_TIMEOUT=300
```

Comment:
Oops... I think I went digging for the correct env var and got distracted halfway thru. :blush: